src/main/java/athena/SortTaskByImportance.java

```diff
@@ -0,0 +1,30 @@
+package athena;
+
+import athena.task.Task;
+
+import java.util.Comparator;
+
+public class SortTaskByImportance implements Comparator<Task> {
```

Comment:
I think we should name it as something that sounds like a `Comparator`, or at least a noun. Something like `TaskImportanceComparator`?